dolphinscheduler-bom/pom.xml

```diff
@@ -982,6 +983,12 @@
                 <artifactId>delight-nashorn-sandbox</artifactId>
                 <version>${nashorn-sandbox.version}</version>
             </dependency>
+
+            <dependency>
+                <groupId>com.orbitz.consul</groupId>
+                <artifactId>consul-client</artifactId>
+                <version>${consul-client.version}</version>
+            </dependency>
```

Comment:
-1 to this.

Since we already have multiple registry plugins, add a new one registry plugin is not a urgent need.

And this repo has been archived. https://github.com/rickfast/consul-client

Reply:
> -1 to this.
> 
> Since we already have multiple registry plugins, add a new one registry plugin is not a urgent need.
> 
> And this repo has been archived. https://github.com/rickfast/consul-client

+1